src/combinator/slice.js

```diff
@@ -159,3 +159,35 @@ SkipWhileSink.prototype.event = function (t, x) {
 
   this.sink.event(t, x)
 }
+
+export function skipAfter (p, stream) {
+  return new stream.constructor(new SkipAfter(p, stream.source))
+}
+
+function SkipAfter (p, source) {
+  this.p = p
+  this.source = source
+}
+
+SkipAfter.prototype.run = function run (sink, scheduler) {
+  return this.source.run(new SkipAfterSink(this.p, sink), scheduler)
+}
+
+function SkipAfterSink (p, sink) {
+  this.p = p
+  this.sink = sink
+  this.skipping = false
+}
+
+SkipAfterSink.prototype.event = function event (t, x) {
+  var p = this.p
+  if (this.skipping) {
+    this.sink.end(t, x)
```

Comment:
This will wait until the next event before ending.  I think it can end immediately after seeing the first value where `p(x)` is true.  How about:

```js
SkipAfterSink.prototype.event = function event (t, x) {
  var p = this.p
  this.skipping = p(x)
  this.sink.event(t, x)

  if (this.skipping) {
    this.sink.end(t, x)
  }
}
```

We may want to add a `if(this.skipping) return` guard at the beginning of the function (similar to the guard in `TakeWhileSink`) as well.